master/usage/configuration/node.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Configuring a node IP address and subnet
+---
+
+Each Calico node has key routing configuration that may be specified in a number of
+different ways.  In particular, this includes the IPv4 and IPv6 addresses and 
+subnets used for routing.
+
+To configure the IP address and subnet of your node, you have the options of:
+-  Set appropriate flags on the `calicoctl node run` command
+-  Set appropriate environments for the `calico/node` container
+-  Configure the node resource using `calicoctl`
+
+> Note that for nodes that were configured prior to v2.1, the subnet information 
```

Comment:
Rather than have a big note here, why don't we just make this one of the reasons you might want to follow this doc (see comment above).

e.g

```markdown
However, there are some scenarios where you may want to configure these values explicitly
rather than use auto-detection:

- You want to change the subnet on the Node to use Calico's [cross-subnet IPIP](...) feature.
```